localstack/services/awslambda/provider.py

```diff
@@ -384,6 +392,55 @@ def _verify_env_variables(env_vars: dict[str, str]):
                 Type="User",
             )
 
+    @staticmethod
+    def _validate_layers(new_layers: list[str]):
+        visited_layers = dict()
+
+        for layer_version_arn in new_layers:
```

Comment:
Shouldn't we also check if the compatible runtimes match if present?

Reply:
Also the max of 5 layers per function?

Reply:
added test todos for these cases